sdk/storage/Azure.Storage.Blobs.Batch/src/BlobBatch.cs

```diff
@@ -159,7 +159,7 @@ private void SetBatchOperationType(BlobBatchOperationType operationType)
         /// <summary>
         /// The <see cref="DeleteBlob(string, string, DeleteSnapshotsOption, BlobRequestConditions)"/>
         /// operation marks the specified blob or snapshot for  deletion. The
-        /// blob is later deleted during garbage collection.
+        /// blob is later deleted during garbage collection which could take up to 30 seconds.
```

Comment:
Instead of making a defined time limit of the SLA of BlobBatchClient.Delete can we reference the documentation where this number is coming from instead? That way in case the SLA changes, we don't have to update our XML docs.

Reply:
Same comment for everywhere else where the SLA is specified

Reply:
I'm unsure of the right doc for the SLA. @seanmcc-msft Are the REST API docs you shared best to link to? And even with them, not all of the docs mention the time.

Any other doc in mind that we could link to?

Reply:
I don't know if the SLA is publicly documented.  It might be better to say "which could take several minutes" to better set expectations.

Reply:
I don't know if the SLA is publicly documented. It might be better to say "which could take several minutes" to better set expectations.

Reply:
@seanmcc-msft in that case, I will update the comments like that.

Reply:
Done 👍